docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/data-import/spark-writer.md

```diff
@@ -267,6 +267,7 @@ Example of a mapping file for the input source:
 #### Spark Properties
 
 The following table gives some example properties, all of which can be found in [Spark Available Properties](http://spark.apache.org/docs/latest/configuration.html#available-properties).
+Note: Only Spark > 2.0 version is supported.
```

Comment:
Not only Spark but also Hadoop, Hive, etc.

Reply:
@darionyaphet Please help. What versions do we need here?